nav2_regulated_pure_pursuit_controller/src/parameter_handler.cpp

```diff
@@ -37,7 +38,13 @@ ParameterHandler::ParameterHandler(
   logger_ = logger;
 
   declare_parameter_if_not_declared(
-    node, plugin_name_ + ".desired_linear_vel", rclcpp::ParameterValue(0.5));
```

Comment:
We should have a migration log here. If `desired_linear_vel` is set, log a warning that it has been changed to `max_linear_vel` and that they should migrate the use.